softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -162,6 +162,28 @@ module.exports = {
       message.setTo(superusers);
       sendMail(message, 'E-Mail', callback);
     });
+  },
+
+  sendMailToContactPersonsOfGroup: function (groupId, message, callback) {
+    const type = '$t(mailsender.notification)';
+    groupsService.getGroups([groupId], (groupLoadErr, groups) => {
+      if (groupLoadErr) { return callback(groupLoadErr, mailtransport.statusmessageForError(type, groupLoadErr)); }
+      if (groups.length !== 1) {
+        const groupNotFoundError = new Error(`${groups.length} Gruppen für Id ${groupId} gefunden. Erwarte genau eine Gruppe.`);
```

Comment:
Was hilft das dem Anwender? - Bitte ändern in etwas actionables wie "Das senden der E-Mail ist fehlgeschlagen. Es liegt ein technisches Problem vor."

Reply:
Eventuell gemeinsam mit nächstem Error behandeln.

Reply:
Und dem nächsten. Hier wäre allerdings noch besser, dann den Dialog nicht zu ermöglichen! :)

Reply:
Wenn das feature in der Gruppe nicht aktiviert ist wir der Link auf der Gruppenseite nicht angezeigt.
Die Überprüfung würde ich aber trotzdem drin lassen, für alle die über curl an der Schnittstelle rumspielen wollen.

Reply:
Wie meinst Du das mit dem Rumspielen? - Die Prüfung, ob man senden darf, ist davon ja nicht berührt. Mir geht es um das finden einer Gruppe nach ID.
Also:
- Gruppe suchen -> ggf. techn Fehler
- Gruppe auf feature prüfen / keine Organizer -> mailsender.contact_persons_cannot_be_contacted

Insbesondere ist ja richtig, dass wenn kein Organizer, dann kann auch nicht gesendet werden! ;)

Reply:
@leider  @signed und mein Punkt bzgl. "mit curl rumspielen" ist, dass der Dialog zwar nicht verlinkt ist, wenn das Feature deaktiviert ist, aber man trotzdem auf die URL einen POST ausführen kann (ich glaube sogar, wenn man die URL für den Dialog kennt, kann man ihn auch aufrufen, wenn das Feature deaktiviert ist). Daher wollten wir das hier nochmal prüfen. Ein Szenario bei dem das sogar relevant wäre:
1. Feature ist gerade aktiviert
1. Nutzer ruft Formular auf
1. es vergeht etwas Zeit, z. B. Nutzer klappt Notebook zu
1. Feature wird deaktiviert
1. Nutzer klappt Notebook z. B. am nächsten Tag wieder auf und nutzt den Dialog zum versendne
1. Mail wird aber nicht versendet, da wir das nochmal abprüfen

Reply:
Durch die Anpassung des Fehlertexts in 3a301c83 war das Zusammenfassen der beiden Fehler nicht mehr so einfach möglich. Dafür wird jetzt dem Nutzer bei keiner oder mehreren gefunden Gruppen für eine Id jetzt nur noch "Das senden der E-Mail ist fehlgeschlagen. Es liegt ein technisches Problem vor." angezeigt und die Details geloggt.